std/complex.d

```diff
@@ -749,13 +749,45 @@ if (is(T R == Complex!R))
 
 
 /**
-   Params: z = A complex number.
-   Returns: The absolute value (or modulus) of `z`.
-*/
+ * Calculates the absolute value (or modulus) of a complex number.
+ *
+ *      $(TABLE_SV
+ *      $(TR $(TH $(I z))                          $(TH abs(z))             $(TH Notes))
+ *      $(TR $(TD (0, 0))                          $(TD 0)                  $(TD ))
+ *      $(TR $(TD (NaN, any) or (any, NaN))        $(TD NaN)                $(TD ))
+ *      $(TR $(TD (Inf, any) or (any, Inf))        $(TD Inf)                $(TD ))
+ *      $(TR $(TD (a, 0))                          $(TD a)                $(TD Direct real absolute value ))
+ *      $(TR $(TD (0, b))                          $(TD b)                $(TD Direct imaginary absolute value ))
+ *      $(TR $(TD (tiny, tiny))                    $(TD max(|re|, |im|))    $(TD When both components are subnormal ))
+ *      $(TR $(TD (a, b)) normal case              $(TD hypot(a, b))      $(TD Uses algorithm to prevent overflow/underflow ))
```

Comment:
The new algorithm has removed the need for some special cases, and fixed the `(tiny, tiny)` case.  So suggest removing them from the table.
```suggestion
 *      $(TR $(TH $(I z))                        $(TH abs(z))           $(TH Notes))
 *      $(TR $(TD (0, 0))                        $(TD 0)                $(TD ))
 *      $(TR $(TD (NaN, any) or (any, NaN))      $(TD NaN)              $(TD ))
 *      $(TR $(TD (Inf, any) or (any, Inf))      $(TD Inf)              $(TD ))
 *      $(TR $(TD (a, b))                        $(TD hypot(a, b))      $(TD Uses algorithm to prevent overflow/underflow ))
```